include/hip/hcc_detail/hip_runtime.h

```diff
@@ -109,9 +109,6 @@ extern int HIP_TRACE_API;
 #include <hip/hcc_detail/device_functions.h>
 #include <hip/hcc_detail/surface_functions.h>
 #include <hip/hcc_detail/texture_functions.h>
-#if __HCC__
-#include <hip/hcc_detail/math_functions.h>
```

Comment:
Does this mean that all HIP applications that use these math functions will also need to update and add `#include <hip/math_functions.h>`?

Reply:
Err no, that's a typo, thank you for spotting it.